packages/grafana-runtime/src/unstable.ts

```diff
@@ -11,3 +11,10 @@
 
 // This is a dummy export so typescript doesn't error importing an "empty module"
 export const unstable = {};
```

Comment:
You can delete this `unstable` export now there is something to actually export. 🙂